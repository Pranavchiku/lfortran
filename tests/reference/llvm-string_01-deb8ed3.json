{
    "basename": "llvm-string_01-deb8ed3",
    "cmd": "lfortran --no-color --show-llvm {infile} -o {outfile}",
    "infile": "tests/../integration_tests/string_01.f90",
    "infile_hash": "e7d4974d8a9b9704bf0c02d7943d916b40c591de3c9e89d30de28d7d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "llvm-string_01-deb8ed3.stdout",
    "stdout_hash": "f0b2b34403e3512bcd5a752221779be3f363d4617f00acae8e449f0c",
    "stderr": "llvm-string_01-deb8ed3.stderr",
    "stderr_hash": "8d501aa6c1185b8a420c56615491d238917e2e2ffb25459e9119640c",
    "returncode": 0
}