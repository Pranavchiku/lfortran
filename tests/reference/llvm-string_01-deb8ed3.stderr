warning: Implied save feature
 --> tests/../integration_tests/string_01.f90:4:1
  |
4 | character(len = 7) :: my_name = 'Dominic'
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
